drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/menu/ScenarioContextMenuRegistry.java

```diff
@@ -49,7 +48,6 @@
     protected GivenContextMenu givenContextMenu;
     protected ExpectedContextMenu expectedContextMenu;
     protected GridContextMenu gridContextMenu;
-    protected UnmodifiableColumnGridContextMenu unmodifiableColumnGridContextMenu;
```

Comment:
@danielezonca @dupliaka Yes, I removed it because its behavior is a copy of `GridContextMenu` in my opinion. 
Please consider, `UnmodifiableColumnGridContextMenu` is shown on grid data cell of the first and second column with this context menu:
![Screenshot from 2020-05-06 16-34-13](https://user-images.githubusercontent.com/16005046/81387213-9f87b300-9116-11ea-86a5-62b9b5933742.png)
And `GridContextMenu` is show in the remaining columns, with this context menu:
![Screenshot from 2020-05-06 16-34-09](https://user-images.githubusercontent.com/16005046/81387345-ce058e00-9116-11ea-90a9-34e1a77478d1.png)
As you can see, the only difference is between the missing "Scenario" title. The items are the same. I think i's correct to have the "Scenario" title for the first columns as well.

Reply:
Ok thanks for the clarification. Probably at the beginning it wasn't like that